scripts/build_date.py

```diff
@@ -0,0 +1,33 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
```

Comment:
As far as I understand the functioning of `mkdocs-gen-files` plugin, it's worth using it to create file copyright.js (or move this function to custom.js and modify it there) on build time (without making changes to working copy).
BTW, bot seems to give useful suggestions about js code.

Reply:
I've updated the script.